RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,94 @@
+> Release Issue Template
+
+# Lotus X.Y.Z Release
+
+We're happy to announce Lotus X.Y.Z...
+
+## 🗺 What's left for release
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+# 🚢 Estimated shipping date
+
+<Date this release will ship on if everything goes to plan (week beginning...)>
+
+## 🔦 Highlights
+
+< top highlights for this release notes >
+
+## Changelog
+
+< changelog generated by bin/mkreleaselog >
+
+## ✅ Release Checklist
+
+First steps:
+
+  - [ ] Fork a new branch (`release/vX.Y.Z`) from `master` and make any further release related changes to this branch. If any "non-trivial" changes get added to the release, uncheck all the checkboxes and return to this stage.
+  - [ ] Prep the changelog
+  - [ ] Bump the version in `version.go` in the `master` branch to `vX.(Y+1).0-dev`.
+  - [ ] Follow the RC release process to cut the first RC.
+    
+Prepping an RC:
+
+- [ ] version string in `build/version.go` has been updated (in the `release/vX.Y.Z` branch).
+- [ ] tag commit with `vX.Y.Z-rcN`
+- [ ] cut a pre-release on [github](https://github.com/filecoin-project/lotus/releases)
+
+Testing an RC:
+
+- [ ] **Stage 0 - Automated Testing**
+  - [ ] Automated Testing
+    - CI: Ensure that all tests are passing.
+        - TODO: Elaborate what tests
+    - Testground (TODO: Infra support to setup, tests themselves need to be written)
+
+- [ ] **Stage 1 - Internal Testing**
+  - [ ] Upgrade our testnet infra (TODO: Elaborate)
+  - [ ] Upgrade our mainnet infra
```

Comment:
Not clear whether we should be upgrading our mainnet infra to an RC, or waiting for the actual release. Feedback welcome @travisperson @coryschwartz @ognots .

Reply:
From travis: There is a subset of mainnet development infra that is a good candidate for RCs to go on, along with a subset of production infra (perhaps some of the bootstrappers).

Reply:
For mainnet infra we want to upgrade a few of the development full archive nodes we have and ensure that any staging deployments that use the lotus api are targeting them.

For the production infra we should upgrade some of the bootstrappers. Current thoughts are running rcs on 1 of the 3 we have in each region.

Reply:
I think it's okay to upgrade an RC to a few nodes. I could also imagine doing this on calibrationnet first and then informing early testers to try calibrationnet.

If we upgraded only calibrationnet first, that would give a chance to do network upgrades along with early adopters and get feedback about the network upgrade before doing it on mainnet. It gives miners and daemon-operators a chance to practice the upgrade.

Reply:
Incorporated the suggestions here